src/common/ini_handler.c

```diff
@@ -1,10 +1,57 @@
 // INI file handler (ini_handler.c)
 
 #include "ini_handler.h"
+#include "ini.h"
 #include "ff.h"
+#include "netif_settings.h"
 
 static const char network_ini_file_name[] = "/lan_settings.ini"; //change -> change msgboxes in screen_lan_settings
 
+static int load_netconfig_handler(void *user, const char *section, const char *name, const char *value) {
```

Comment:
this does not belong to ini stuffs, also if possible pass a pointer to structure than four pointers

Reply:
Ok, i will move it to netif_settings. Passing pointer to structure is not possible. Parameter of this function are predefined in ini.h